conf/kafka-hdfs-pipeline.conf.template

```diff
@@ -0,0 +1,43 @@
+gearpump {
```

Comment:
I think the conf should go to `src/main/resources/reference.conf` while kafka config file has been removed after the refactoring